### PR TITLE
feat: Created Input component

### PR DESCRIPTION
**Pull request by lakshaychopra**

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    width: 100%;
+}
+
+.input-group input,
+textarea {
+    width: 80%;
+    padding: 0.4rem;
+    font-size: 0.8rem;
+    border-radius: 5px;
+    border: 1px solid black;
```
Comment (someshtiwarii) [inline components/input/input.css:14]: can use color variable

File: css/docsite.css [hunk 2]
```diff
@@ -36,6 +50,19 @@
     gap: 1rem;
 }
 
+.main-section .example-row-start {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: flex-start;
+    background-color: #f8f9fa;
```
Comment (someshtiwarii) [inline css/docsite.css:58]: can use color variable

**Review by someshtiwarii — APPROVED** [review]
apart from the mentioned , rest looks good to me

**Review by shagunbagrecha — APPROVED** [review]
Looks good!

**Review by himadri2110 — APPROVED** [review]
Looks good!

**Pull request merged**